_data/sites.json

```diff
@@ -3619,6 +3619,18 @@
         ]
     },
 
+    {
+        "name": "CEX.IO",
```

Comment:
Shouldn't it be `cfx.re`?

Reply:
Yes indeed, I mistyped

Reply:
No, I did not mistype, just forgot to change the title